### PR TITLE
docs: fix snippets using jsdoc-region-tag

### LINKED ISSUE #566: Snippets are missing from API reference docs.
@bcoe created a new plugin jsdoc-region-tag.

Snippets are missing from API reference docs and was surfaced by @billyjacobson.

### PR DESCRIPTION
**Pull request by frankyn**
Adding jsdoc-region-tag. Also found a region tag issue and fixed it while I was adding the plugin.

Staged documentation can be found at: https://storage.googleapis.com/coderfrank-staged-docs/bigtable/docs/Bigtable.html

- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #566🦕

### CODE REVIEW

**Comment by frankyn** [comment]
@billyjacobson could you review the staged documentation for Bigtable? The link is in the PR description.

@JustinBeckwith, I didn't add new files, but I'm getting a header-check issue. The license header is the same for other files. Is this a bug or is this something I should fix?

**Comment by JustinBeckwith** [comment]
@frankyn you can safely ignore that warning.

File: synth.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 templates = common_templates.node_library(
   source_location='build/src'
 )
-s.copy(templates)
+s.copy(templates, exclude='.jsdoc.js')
```
Comment (bcoe) [inline synth.py]: whoops this should be:

```
excludes=['.jsdoc.js`]
```

I'll make an effort to remove the necessity of this step early this week.

Comment (frankyn) [reply synth.py]: My bad, thanks @bcoe! Sending a fix now.

**Comment by billyjacobson** [comment]
Looks great, thanks so much

**Pull request merged**